### PR TITLE
Add alpine-keys to sdk image

### PR DESCRIPTION
**Pull request by jdolitsky**

### CODE REVIEW

File: images/sdk/configs/latest.apko.yaml [hunk 1]
```diff
@@ -6,7 +6,6 @@ contents:
     - '@local /github/workspace/packages'
   packages:
     - sdk@local
-    - wolfi-baselayout
```
Comment (luhring) [inline images/sdk/configs/latest.apko.yaml:9]: Why remove this line? (I had thought we want to include this line in all images)

Comment (jdolitsky) [reply images/sdk/configs/latest.apko.yaml:9]: I simply pushed it into the melange file as a dependency of sdk

**Review by dlorenc — APPROVED** [review]

**Comment by jdolitsky** [comment]
```
Jan  5 22:13:00.660 [DEBUG] [arch:x86_64] [cmd:apk] [use-proot:false] [use-qemu:] fetch https://packages.wolfi.dev/os/x86_64/APKINDEX.tar.gz
Jan  5 22:13:00.895 [DEBUG] [arch:x86_64] [cmd:apk] [use-proot:false] [use-qemu:] ERROR: unable to select packages:
Jan  5 22:13:00.898 [DEBUG] [arch:x86_64] [cmd:apk] [use-proot:false] [use-qemu:]   alpine-keys (no such package):
Jan  5 22:13:00.898 [DEBUG] [arch:x86_64] [cmd:apk] [use-proot:false] [use-qemu:]     required by: sdk-0.0.1-r0[alpine-keys]
Error: failed to build layer image: initializing apk: failed to fixate apk world: exit status 2
2023/01/05 22:13:00 error during command execution: failed to build layer image: initializing apk: failed to fixate apk world: exit status 2
```

looks like wolfi package not yet avail

**Comment by jdolitsky** [comment]
closing in favor of #197

**Pull request closed without merging**